index.js

```diff
@@ -190,7 +194,7 @@ module.exports = (bookshelf, options = {}) => {
 
         return model.historyOptions.model
           .forge(forge)
-          .query(qb => qb.orderBy('sequence', 'desc'))
+          .query(qb => qb.orderBy(fields.sequence, 'desc'))
```

Comment:
This fixes a bug where we need the configured `sequence`, not the hard coded value.